<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>792</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Soccer League Manager</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView_player_picture">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>53</y>
      <width>134</width>
      <height>174</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_first_name">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>63</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>55</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>First Name</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_last_name">
    <property name="geometry">
     <rect>
      <x>375</x>
      <y>63</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Last Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_team">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>113</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Team</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_dob">
    <property name="geometry">
     <rect>
      <x>375</x>
      <y>113</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Date of Birth</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_first_name">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>83</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_last_name">
    <property name="geometry">
     <rect>
      <x>375</x>
      <y>83</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_select_team">
    <property name="geometry">
     <rect>
      <x>179</x>
      <y>133</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Select Team</string>
     </property>
    </item>
   </widget>
   <widget class="QDateEdit" name="dateEdit_dob">
    <property name="geometry">
     <rect>
      <x>375</x>
      <y>133</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_players">
    <property name="geometry">
     <rect>
      <x>597</x>
      <y>0</y>
      <width>601</width>
      <height>741</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>601</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>601</width>
      <height>741</height>
     </size>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <property name="rowCount">
     <number>19</number>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>First Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Last Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Team</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Date of Birth</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Picture</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Active</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_add_picture">
    <property name="geometry">
     <rect>
      <x>62</x>
      <y>235</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Picture</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_add_player">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>2</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Icons/user-plus.png</normaloff>../Icons/user-plus.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_remove_player">
    <property name="geometry">
     <rect>
      <x>501</x>
      <y>2</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Icons/user-minus.png</normaloff>../Icons/user-minus.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_save">
    <property name="geometry">
     <rect>
      <x>532</x>
      <y>2</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Icons/save.png</normaloff>../Icons/save.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_print">
    <property name="geometry">
     <rect>
      <x>563</x>
      <y>2</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Icons/printer.png</normaloff>../Icons/printer.png</iconset>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_search">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>2</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_search">
    <property name="geometry">
     <rect>
      <x>375</x>
      <y>2</y>
      <width>93</width>
      <height>23</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionPrint"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuAdd">
     <property name="title">
      <string>Add..</string>
     </property>
     <addaction name="actionAdd_Player"/>
     <addaction name="actionAdd_Team"/>
    </widget>
    <widget class="QMenu" name="menuRemove">
     <property name="title">
      <string>Remove..</string>
     </property>
     <addaction name="actionRemove_Player"/>
     <addaction name="actionRemove_Team"/>
    </widget>
    <addaction name="menuAdd"/>
    <addaction name="menuRemove"/>
    <addaction name="actionSearch"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionAdd_Player">
   <property name="text">
    <string>Add Player</string>
   </property>
  </action>
  <action name="actionRemove_Player">
   <property name="text">
    <string>Remove Player</string>
   </property>
  </action>
  <action name="actionRemove_Team">
   <property name="text">
    <string>Remove Team</string>
   </property>
  </action>
  <action name="actionAdd_Team">
   <property name="text">
    <string>Add Team</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="text">
    <string>Search..</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
